### PR TITLE
Avoid analytics exception when disabled

### PR DESCRIPTION
**Pull request by bgrozev**
Prevent sendEvent from executing when the analytics are disposed (which happens unless handlers are configured).

### CODE REVIEW

**Comment by paweldomas** [comment]
please create a branch in jitsi repo next time - it's such a PITA having to add a new remote just to check a PR...

**Comment by bgrozev** [comment]
You don't have to add a new remote, github PRs are available in the github
repo. You might need to add a line to your .git/config to fetch them, but
you do that only once (I'll post the config later).

Boris

**Comment by bgrozev** [comment]
https://help.github.com/articles/checking-out-pull-requests-locally/
`git fetch origin pull/ID/head:BRANCHNAME`

Jenkins test this please

**Comment by bgrozev** [comment]
Jenkins test this please

**Comment by bgrozev** [comment]
Jenkins test this please

**Comment by damencho** [comment]
There is a different version of flow between lib-jitsi-meet and jitsi-meet.

> Wrong version of Flow. The config specifies version ^0.56.0 but this is version 0.57.3

Maybe you need to rebase to latest ...

**Comment by bgrozev** [comment]
> Maybe you need to rebase to latest ...

Thanks!

File: modules/RTC/ScreenObtainer.js [hunk 1]
```diff
@@ -295,7 +295,8 @@ const ScreenObtainer = {
                     if (firefoxExtInstalled === null) {
                         firefoxExtInstalled = false;
                     }
-                    this.obtainScreenOnFirefox(callback, errorCallback);
+                    this.obtainScreenOnFirefox(
+                        options, callback, errorCallback);
```
Comment (virtuacoplenny) [inline modules/RTC/ScreenObtainer.js]: What is this change about?

Comment (virtuacoplenny) [reply modules/RTC/ScreenObtainer.js]: I see now it's in a separate commit.

**Pull request merged**